content/blog/2025-02-03-submit_posit_conf/index.en.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Tips for Submitting a Video Abstract to Posit Conf 2025"
+author: 
+  - Nic Crane
+  - Sayantika Banik
+date: '2025-02-03'
+---
+
+Thinking about submitting a talk for **Posit Conf 2025** but feeling nervous about recording a video abstract? You're not alone! It can feel intimidating at first, but the good news is—it's just **one minute** and all about showcasing your idea effectively.
+
+## **Tips from the R-Ladies Global Team**
+
+We know that recording a video can feel like a challenge, so we asked members of **R-Ladies Global** to share their best advice:
+
+📽️  **Shannon Pileggi**: Ask more than one person for feedback on your abstract and video content - everyone has a different and valuable point of view.Think about the way the principles of your content apply broadly to a general audience. [[Read Shannon's blog post talking about her experience of getting expected to speak at Posit Conf here](https://www.pipinghotdata.com/posts/2024-10-24-iterating-to-achieve-my-first-accepted-positconf-talk/)]
```

Comment:
```suggestion
📽️  **Shannon Pileggi**: Ask more than one person for feedback on your abstract and video content - everyone has a different and valuable point of view.Think about the way the principles of your content apply broadly to a general audience. [[Read Shannon's blog post talking about her experience of getting accepted to speak at Posit Conf here](https://www.pipinghotdata.com/posts/2024-10-24-iterating-to-achieve-my-first-accepted-positconf-talk/)]
```

Reply:
Oh god, that's ADHD brain stuff right there, thanks!